core-web/libs/portlets/edit-ema/portlet/src/lib/store/features/editor/withEditor.ts

```diff
@@ -55,9 +56,9 @@ const buildIframeURL = ({ pageURI, params, isTraditionalPage }) => {
         return new String('');
     }
 
-    const pageAPIQueryParams = createPageApiUrlWithQueryParams(pageURI, params);
-    const origin = params.clientHost || window.location.origin;
-    const url = new URL(pageAPIQueryParams, origin);
+    const queryParams = normalizeQueryParams(params);
+    const pageAPIQueryParams = buildFullPageURL(pageURI, queryParams);
+    const url = new URL(pageAPIQueryParams, params.clientHost || window.location.origin);
```

Comment:
should we jus have a `getUrl` type of thing.

Reply:
Done!